net/dns/resolvconf.go

```diff
@@ -2,20 +2,28 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build linux freebsd
-
 package dns
 
 import (
 	"bufio"
 	"bytes"
+	_ "embed"
 	"fmt"
 	"os"
 	"os/exec"
 
+	"tailscale.com/atomicfile"
 	"tailscale.com/types/logger"
 )
 
+//go:embed resolvconf-workaround.sh
+var legacyResolvconfScript []byte
+
+// resolvconfConfigName is the name of the config submitted to resolvconf.
+// It has this form to match the "tun*" rule in interface-order
+// when running resolvconfLegacy, hopefully placing our config first.
+const resolvconfConfigName = "tun-tailscale.inet"
```

Comment:
what's the meaning of this string literal value? Does tun-* do something? Does .inet do something?